### PR TITLE
Profile screen: move header from tableView to separate view

### PR DESCRIPTION
**Pull request by spanastasia**

### CODE REVIEW

**Review by oleksiikolomiietssnapp — APPROVED** [review]

File: INTITA/Modules/Profile/ProfileViewController.swift [hunk 1]
```diff
@@ -7,19 +7,22 @@
...
+    @IBOutlet weak var headerView: UIView!
+    lazy var headerViewContent: ProfileHeaderViewCell = .fromNib()
+    
     override var preferredStatusBarStyle: UIStatusBarStyle { .lightContent }
     
-    //MARK: - View life cycle
+    //MARK: - View lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        colorStatusBar(UIColor.primaryColor)
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/ProfileViewController.swift:22]: This one doesn't conflict with the white screen in the bottom below the footer now?

File: INTITA/Modules/Profile/ProfileViewController.swift [hunk 2]
```diff
@@ -30,98 +33,87 @@ class ProfileViewController: UITableViewController, Storyboarded {
         tableView.dataSource = self
         tableView.delegate = self
         registerCells()
+        headerViewContent.delegate = coordinator
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/ProfileViewController.swift:36]: Here the coordinator push .xib header or what is his role?

Comment (spanastasia) [reply INTITA/Modules/Profile/ProfileViewController.swift:36]: it will push another viewController (after it will be created)

Comment (svitlana-korostelova) [reply INTITA/Modules/Profile/ProfileViewController.swift:36]: do you show 2 controllers - tableview and header at the same time?

Comment (spanastasia) [reply INTITA/Modules/Profile/ProfileViewController.swift:36]: no, it has only one controller ProfileViewController
this delegate is for reacting by tapping on the avatar or edit button. When the user taps, headerViewContent says to its delegate to show SystemProfileViewController (but it doesn't exist yet, that's why we show an alert with "Oops..")

Comment (svitlana-korostelova) [reply INTITA/Modules/Profile/ProfileViewController.swift:36]: ok

**Pull request merged**